AGPs/AGP-Comprehensive-Aragon-Bug-Bounty-Program.md

```diff
@@ -0,0 +1,56 @@
+---
+AGP: N/A
+Title: Comprehensive Aragon Bug Bounty Program
+Author: burrrata (@burrrata)
+Status: Stage III
+Track: Association
+Created: 2019-10-02
+---
+
+# AGP-X: Comprehensive Aragon Bug Bounty Program
+
+## Description of desired Association policy change
+
+This proposal is to create a comprehensive [Bug Bounty Program](https://wiki.aragon.org/dev/bug_bounty/) for all funded Aragon applications and tooling. 
+
+### Bug Bounty Scope
+
+The scope of the [Aragon Association Bug Bounty Program](https://wiki.aragon.org/dev/bug_bounty/) will be expanded to make all Nest grantees and Flock team applications eligible. Applications will only be accepted into the bug bounty program if they have undergone a security audit, incorporated all major and critical security audit recommendations, and shipped to mainnet. 
```

Comment:
Comment by @izqui 

> We will need to increase the scope to include the Aragon Court soon, I would preemptively add it to the list even though the program won't launch until A1 considers de codebase ready for it.

https://github.com/aragon/AGPs/pull/107#discussion_r333364180

Reply:
I would also consider adding all apps that go through the AA's Security Review process, but keep in mind the app's install rate for the CVSS classification's `availability` score.

Reply:
@light I believe this is covered as A1 is a Flock team, however in the new version I've added a comment specifically mentioning the Aragon Court

@sohkai That's an excellent idea, however at the moment I don't believe we have a standard method to track usage. Once that is established, then I hope this AGP is modified accordingly.

Reply:
> we have a standard method to track usage

I proposed an app's install rate as part of the availability score, because if only one org has installed a vulnerable app, no matter how bad it is, the surface area is small (for the entire Aragon Network).